src/action-ros-ci.ts

```diff
@@ -304,16 +304,24 @@ async function run() {
 					colconCommandPrefix += `source ${ros2SetupPath} && `;
 				}
 			}
+		} else if (isWindows) {
+                        // Windows only supports ROS2
+			if (targetRos2Distro) {
+                                const ros2SetupPath = `C:\dev\${targetRos2Distro}\local_setup.bat`;
+				if (fs.existsSync(ros2SetupPath)) {
+					colconCommandPrefix += `call ${ros2SetupPath} && `;
+				}
+			}
 		}
-
+		
 		let colconBuildCmd = `colcon build --event-handlers console_cohesion+ \
 			--packages-up-to ${packageNameList.join(" ")} \
 			${extra_options.join(" ")} \
 			--cmake-args ${extraCmakeArgs}`;
 		if (!isWindows) {
 			colconBuildCmd = colconBuildCmd.concat(" --symlink-install");
 		}
-		await execBashCommand(colconBuildCmd, colconCommandPrefix, options);
+		await execBashCommand(colconBuildCmd, colconCommandPrefix, options, colconCommandPrefix);
```

Comment:
Is this `log_message` argument intentional? If you omit this argument, then `execBashCommand` will print the entire command instead of just the prefix

Reply:
oh i just put that in there because from my tests it looks like that the colconCommandPrefix isnt being added to the command. maybe i missed it but it looks like for windows it isnt adding it in.